modules/rostests/kmtests/fltmgr/fltmgr_create/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+include_directories(../../include)
+
+list(APPEND FLTMGR_TEST_DRV_SOURCE
+    ../../kmtest_drv/kmtest_fsminifilter.c
+    fltmgr_create.c)
+
+add_library(fltmgr_create SHARED ${FLTMGR_TEST_DRV_SOURCE})
+set_module_type(fltmgr_create kernelmodedriver)
+target_link_libraries(fltmgr_create kmtest_printf ${PSEH_LIB})
+add_importlibs(fltmgr_create ntoskrnl fltmgr hal)
```

Comment:
hal and ntoskrnl should generally be last in the imports list.

Reply:
Thanks, I'll fix that in the next commit